ComponentKit/Core/CKComponent.h

```diff
@@ -54,4 +54,12 @@ struct CKComponentViewContext {
  */
 - (id)nextResponder;
 
+/**
+ When an event occurs, if a component implements the canPerformAction method it will be called in the normal responder
```

Comment:
Consider:

"When an action is triggered, a component may use this method to either capture, or ignore a given action.  The default implementation simply uses respondsToSelector: to determine if the component can perform the given action.

In practice this is useful only for integrations with UIMenuController, whose API walks the UIResponder chain to determine which menu items to display.  You should not override this for standard component actions."